tabbycat/api/views.py

```diff
@@ -26,6 +29,38 @@ class AdministratorAPIMixin:
     permission_classes = [IsAdminUser]
 
 
+class APIRootView(AdministratorAPIMixin, GenericAPIView):
```

Comment:
To go with the `/tournaments/` prefix, the API root should just be similar to:
```json
{
  "_urls": {
    "tournaments": "/tournaments"
  }
}
```
As just "directions" to the tournaments (or institutions with that side-discussion).

Reply:
I agree.